azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -518,45 +424,115 @@ private static void loadViewerPlugins(final Context root, final String pluginPat
     ve.addProperty("jar.resource.loader.path", jarResourcePath);
   }
 
-  /**
-   * Loads the Azkaban property file from the AZKABAN_HOME conf directory
-   */
-  private static Props loadConfigurationFromAzkabanHome() {
```

Comment:
This method is not used and has been removed. The code below is copy paste from above except that the static modifier has been removed and references have been updated accordingly.